CFE-635: If the profiling is enabled on the CRI-O service, machine config resource does not get created (#51767)
openshift/openshift-docs: modules/node-observability-create-custom-resource.adoc

[IMPORTANT]
====

> If the CRI-O profiling is not enabled on the worker nodes, the `NodeObservabilityMachineConfig` resource gets created and the worker nodes matching the `nodeSelector` specified in `NodeObservability` CR get rebooted. This might take 10 or more minutes to complete.

Reviewer: "rebooted" is a use when necessary word. I think you could actually use restart here. So:
```suggestion
If the CRI-O profiling is not enabled on the worker nodes, the `NodeObservabilityMachineConfig` resource gets created and the worker nodes matching the `nodeSelector` specified in `NodeObservability` CR restarts. This might take 10 or more minutes to complete.
```
See the IBM style for reference https://www.ibm.com/docs/en/ibm-style?topic=word-usage#r

> If the CRI-O profiling is not enabled on the worker nodes, the `NodeObservabilityMachineConfig` resource gets created and the worker nodes matching the `nodeSelector` specified in `NodeObservability` CR get rebooted. This might take 10 or more minutes to complete.

Reviewer: Just a suggestion to break that first sentence into two for clarity and tense.
```suggestion
If CRI-O profiling is not enabled on the worker nodes, the `NodeObservabilityMachineConfig` resource gets created. Worker nodes matching the `nodeSelector` specified in `NodeObservability` CR will reboot. This might take 10 or more minutes to complete.
```

Author: Addressed

> If the profiling is enabled on the CRI-O service, machine config resource does not get created upon the creation of the `NodeObservability` CR. If the profiling is not enabled, the machine config resource gets created and the worker nodes matching the nodeSelector get rebooted. This might take 10 or more minutes to complete.

Reviewer 2: Minor nit: nodeSelector --> `nodeSelector`

Author: Acked

> If the profiling is enabled on the CRI-O service, machine config resource does not get created upon the creation of the `NodeObservability` CR. If the profiling is not enabled, the machine config resource gets created and the worker nodes matching the `nodeSelector` get rebooted. This might take 10 or more minutes to complete.

Author: sounds good to me. Addressed
